zkml/src/layers/transformer/layernorm.rs

```diff
@@ -62,6 +289,77 @@ impl LayerNorm<f32> {
     }
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct LayerNormCtx {
+    node_id: NodeId,
+    /// The result of calling [`f32::to_bits`] on the epsilon used for normalisation purposes
+    eps: u32,
+    /// The number of bits that get range checked (so we can know how many instances there are in the range lookup)
+    range_check_bits: usize,
+    /// The size of the dimension we normalise over (unpadded)
+    dim_size: usize,
+    /// The multiplier used to scale up inputs to the lookup table.
+    multiplier: Element,
+    /// The base 2 logarithm of the multiplier for the most significant chunk we range check
+    top_chunk_scalar_log: usize,
+}
+
+impl LayerNormCtx {
+    fn calculate_lt_poly_evals<E: ExtensionField>(unpadded_shape: &Shape, point: &[E]) -> Vec<E> {
```

Comment:
Can you elaborate a tidy bit on this one ? In the general sense that i'm not sure what this polynomial refers to in your spec. This is just the evaluation and the construction comes below but still, unclear the role and were it fits for me at least.